config/dev.exs

```diff
@@ -1 +1,9 @@
 use Mix.Config
+
+config :assembly, Assembly.Repo,
```

Comment:
We should setup the db to create uuid primary keys. Then we can just delete the hal id when we are ready and everything _just works_ ™️